lib/resourceusage/usage.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resourceusage
+
+import (
+	"context"
+	"time"
+
+	"github.com/gravitational/trace"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/gravitational/teleport/api/types"
+	apievents "github.com/gravitational/teleport/api/types/events"
+	"github.com/gravitational/teleport/lib/events"
+)
+
+// GetAccessRequestMonthlyUsage returns the number of access requests that have been created this month.
+func GetAccessRequestMonthlyUsage(ctx context.Context, alog events.AuditLogger, now time.Time) (int, error) {
```

Comment:
Test this package, since we now export it?

Reply:
Hm, so this is essentially covered by `lib/auth/usage_test.go` (which uses `GetAccessRequestMonthlyUsage` as an implementation detail). With this suggestion + https://github.com/gravitational/teleport.e/pull/1930#discussion_r1287460569, we would basically end up testing this same logic thrice. 🤔

The conventional solution would be to pull this out into its own interface or similar so it can be mocked out, and then make all of these 3 into true unit tests, but somehow this feels like a bit of an overkil in this case.

Reply:
> The conventional solution would be to pull this out into its own interface or similar so it can be mocked out

-1 to this idea, I'd rather test using the real thing.